plugins/catalog-backend-module-gcp/README.md

```diff
@@ -0,0 +1,3 @@
+# Catalog Backend Module for GCP
+
```

Comment:
Mistaken commit?

Reply:
I made this one on top of our other branch https://github.com/backstage/backstage/pull/18759, this should disappear when the other PR is merged